topics/computational-chemistry/tutorials/covid19-docking/tutorial.md

```diff
@@ -0,0 +1,342 @@
+
+With the example you will find in the `tutorial.bib` file, you can add a citation to
+this article here in your tutorial like this:
+{% raw %} `{% cite Batut2018 %}`{% endraw %}.
+This will be rendered like this: {% cite Batut2018 %}, and links to a
+[bibliography section](#bibliography) which will automatically be created at the end of the
+tutorial. -->
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Background
+
+Early in March 2020, the Diamond Light Source recently completed a successful fragment screen on MPro, which provided 55 fragment hits (see their [press release](https://www.diamond.ac.uk/covid-19/for-scientists/Main-protease-structure-and-XChem.html) here). In an effort to identify candidate molecules for binding, InformaticsMatters, the XChem group and the European Galaxy team joined forces to construct and execute a Galaxy workflow for performing and evaluating molecular docking on a massive scale.
+
+An initial list of 41,000 candidate molecules was assembled by using the Fragalysis fragment network to elaborate from the initial fragment hits, as described [here](https://diamondlightsource.atlassian.net/wiki/spaces/FRAG/pages/8323192/The+Astex+Fragment+network). These were used as inputs for the docking and scoring workflow. The workflow consists of the following steps, each of which was carried out using tools installed on the European Galaxy server:
+1. Charge enumeration of the 41,000 candidate molecules selected based on the fragment hits.
+2. Generation of 3D conformations based on SMILES strings of the candidate molecules.
+3. Docking of molecules into each of the MPro structures using rDock.
+4. Evaluation of the docking poses using a deep learning approach [3] developed by the XChem group and collaborators.
+
+The original study required almost 20 years of CPU time, not counting GPU resources consumed. This is obviously not reproducible as a tutorial. Therefore, we will repeat the workflow with a small library of just 100 molecules, on a single MPro fragment structure. Links will be provided to all the original Galaxy histories, with notes to explain where and why things were done differently to the tutorial.
+
+ <!-- employing augmentation of training data with incorrectly docked ligands to prompt the model to learn from protein-ligand interactions. The algorithm was deployed on the European Galaxy server inside a Docker container, thanks to work by InformaticsMatters and the European Galaxy team. -->
+
+
+# Get data
+
+We require four datasets for the simulation and analysis:
+1. A list of 100 ligand candidates. These are the molecules which will be docking into the protein binding site.
+2. A PDB file of the receptor MPro protein (without ligand or solvent).
+3. A list of fragment hits (17 in total) in SDF format.
+4. A 'Frankenstein ligand', which combines the features of all the fragment hits. This will be used to construct a model of the active site for docking.
```

Comment:
we might be able to create this ligand as well ...

Reply:
you mean, write a tool using this perl script Tim mentions on his website?

Reply:
Yes, seems useful, isn't it and one less input. But this can be done later.

Reply:
will probably leave this for now if it's okay? I'll update the tutorial when the new tool is created

Reply:
Okay.